cmake/ctest/drivers/geminga/cron_driver.sh

```diff
@@ -39,20 +39,26 @@ export TDD_HTTPS_PROXY=$https_proxy
 SCRIPT_DIR=`cd "\`dirname \"$0\"\`";pwd`
 
 
-export export MODULEPATH=$SCRIPT_DIR:$MODULEPATH
+# Trilinos source repo
+export TRILINOS_SOURCE=$SCRIPT_DIR/../../../..
+
+# folder with the machine specific build info
+export BUILDS_DIR=$TRILINOS_SOURCE/cmake/ctest/drivers/$HOSTNAME
+
+# So we can load the env module specific to the build
+export MODULEPATH=$BUILDS_DIR:$MODULEPATH
+echo "Loading modules from $MODULEPATH"
+
+# update the Trilinos source
+pushd $TRILINOS_SOURCE
+./cmake/tribits/python_utils/gitdist --dist-no-color fetch
```

Comment:
Why use `gitdist` instead of `git`?

Reply:
My understanding is that it also updates sub-repos. Not sure if we need that.